Python/pythonrun.c

```diff
@@ -267,6 +285,13 @@ PyRun_InteractiveOneObjectEx(FILE *fp, PyObject *filename,
         }
         return -1;
     }
+
+    if (PyRun_IsInteractiveExitCommand(mod)) {
+        PyErr_SetString(PyExc_SystemExit, "bye!");
```

Comment:
Does this need a string? Typing `exit` doesn't print anything, it just closes the REPL.

Reply:
This is just to show that ir works :)